docs/source/tutorials/arch_modeling/user_defined_temp_tutorial.rst

```diff
@@ -167,4 +167,4 @@ Finally, rerun this command from the OpenFPGA root directory to ensure it is wor
    python3 openfpga_flow/scripts_run_fpga_task.py fpga_verilog/adder/hard_adder --debug --show_thread_logs
 
 
-.. _user_defined_template.v: https://openfpga--274.org.readthedocs.build/en/274/manual/fpga_verilog/fabric_netlist/#cmdoption-arg-user_defined_templates.v
+.. _user_defined_template.v: https://openfpga.readthedocs.io/en/master/manual/fpga_verilog/fabric_netlist/#cmdoption-arg-user_defined_templates.v
```

Comment:
I am currently using this link only at LINE88 (at the beginning of Fixing the Error with user_defined_template.v). Do you think it would be better to have this link in several places (beginning, middle, and end)?

Reply:
Yes. If the link is directed to a page inside the documentation, we should use :ref: to avoid broken links in future (URL may change but :ref: will not). Also, as a tutorial about ``user_defined_template.v``, a link to the detailed documentation is strongly needed. You can even insert the link at the beginning of the tutorial and guide reader to look into it.